### PR TITLE
Added indicator for active clock

### PR DESCRIPTION
**Pull request by seanmtracey**
Clock will be dynamically created if Big FT detects it's not in an already displayed timezone

Removed timezone identifier from the header

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ const __bigFT = (function (){
 
 	const newsTicker = $('.ticker').ticker();
 	const mediaHolder = document.getElementsByClassName('main-stories')[0];
-	const clocks = document.querySelectorAll('[data-tz]');
+	let clocks = undefined;
```
Comment (seanmtracey) [inline src/main.js:83]: The amount of clocks can change, so this needs to be assigned after all required clocks have been generated and added to the DOM

**Comment by seanmtracey** [comment]
Addresses #47

**Comment by seanmtracey** [comment]
indicator

**Comment by JakeChampion** [comment]
@railsagainstignorance @triblondon @AdaRoseEdwards Opinions on this?

**Comment by JakeChampion** [comment]
LGTM :+1:

File: src/main.js (hunk 1 above)
Comment (JakeChampion) [inline src/main.js:83]: :+1:

**Comment by railsagainstignorance** [comment]
works for me in London.
have not tested in other listed TZs or in a non-listed TZ
LGTM

**Pull request merged**